Formula/p/python-psutil.rb

```diff
@@ -0,0 +1,29 @@
+class PythonPsutil < Formula
+  desc "Cross-platform lib for process and system monitoring in Python"
+  homepage "https://github.com/giampaolo/psutil"
+  url "https://files.pythonhosted.org/packages/d6/0f/96b7309212a926c1448366e9ce69b081ea79d63265bde33f11cc9cfc2c07/psutil-5.9.5.tar.gz"
+  sha256 "5410638e4df39c54d957fc51ce03048acd8e6d60abc0f5107af51e5fb566eb3c"
+  license "BSD-3-Clause"
+
+  depends_on "python@3.10" => [:build, :test]
```

Comment:
Unless we have actual uses for it in old python versions, I'd say we keep it to current version.

Reply:
make sense, let me switch to use the latest version.